nibe/coil.py

```diff
@@ -20,8 +24,8 @@ def is_coil_boolean(coil):
 class Coil:
     """Represents a coil."""
 
-    mappings: Optional[Dict[str, str]]
-    reverse_mappings: Optional[Dict[str, str]]
+    mappings: Optional[Dict[str, str]] = None
+    reverse_mappings: Optional[Dict[str, str]] = None
```

Comment:
I think we should set these to empty dict in init instead of having Optional. `if mappings` will work the same for empty dict. But let's do that in a separate change.